ansible/roles/windows_universal_forwarder/tasks/main.yml

```diff
@@ -6,7 +6,7 @@
 - include: collect_attack_simulation_logs.yml
 - include: collect_sysmon_logs.yml
 - include: collect_windows_event_logs.yml
-- include: install_nxlog.yml
+#- include: install_nxlog.yml
```

Comment:
if we are about to comment code, better remove it. git keeps the history of changes.

Reply:
yeah we can remove it. It's better.